src/components/TreeMenu/index.js

```diff
@@ -42,9 +41,10 @@ const TreeMenu = ({
 							/>}
 						</InputGroup>
 						<ListGroup>
-							{items.map(({ reset, ...props }) => (
+							{items.map(({ reset, ...params }) => (
 								<TreeMenuItem
 									active="false"
+									{...params}
```

Comment:
@aringocode Anyway, are the `...params` used anywhere? I dont see the being used in the `TreeMenuItem` so I suppose you want to pass the `{...params}` to the `<li>` or somewhere, not to loose TreeMenuItem behavior

Reply:
What was originally passed to the TreeMenuItem when using just `{...props}` ?

Reply:
@aringocode Well, anyway, I just wanted some information, nevertheless, your previous solution with passing `{...props}` together with `{...params}` was ok. What I wanted was just answer to my questions, why did you choose this solution and if it is propely handled in the `<TreeMenuItem />` component :)